MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionQtTabCalib.ui

```diff
@@ -168,32 +168,30 @@
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
-        <item row="0" column="0">
-         <widget class="QLabel" name="label_new_vanadium_num">
-          <property name="text">
-           <string>Vanadium #:</string>
-          </property>
-         </widget>
-        </item>
         <item row="0" column="1" colspan="2">
-         <widget class="QLineEdit" name="lineEdit_new_vanadium_num">
-          <property name="text">
-           <string>236516</string>
+         <widget class="MantidQt::MantidWidgets::MWRunFiles" name="lineEdit_new_vanadium_num" native="true">
+          <property name="text" stdset="0">
+           <string>241391</string>
```

Comment:
I checked this, and it would be good to keep the old value (236516) - that one is a recent Vanadium run.
241391 is a Calibration/Ceria run, so putting that number there could confuse users a lot.